src/components/dashboard/index.jsx

```diff
@@ -4,40 +4,67 @@ import LineChart from '../common/lineChart'
 import Card from './card'
 import { Navbar, Nav, NavDropdown, Container, Row, Col } from 'react-bootstrap'
 // import { LineChart } from './lineChart'
+import './card.css'
 
 export default class Dashboard extends Component {
   state = {}
 
   render() {
+    const indicator = document.querySelector('.nav-indicator')
+    const items = document.querySelectorAll('.nav-item')
+
+    const handleIndicator = (el) => {
+      items.forEach((item) => {
+        item.classList.remove('is-active')
+        item.removeAttribute('style')
+      })
+
+      indicator.style.width = `${el.offsetWidth}px`
+      indicator.style.left = `${el.offsetLeft}px`
+      indicator.style.backgroundColor = el.getAttribute('active-color')
+
+      el.classList.add('is-active')
+      el.style.color = el.getAttribute('active-color')
+    }
+    items.forEach((item, index) => {
+      item.addEventListener('click', (e) => {
+        handleIndicator(e.target)
+      })
+      item.classList.contains('is-active') && handleIndicator(item)
+    })
```

Comment:
Toggling classname logic can be simplified by storing the active element's id in the component's state and using ternary conditional operator in the element's attribute (inside the return)